### PR TITLE
add comments to statements

### PR DESCRIPTION
**Pull request by Jim8y**
This pr adds comments to the statements convert. Part of my plan to comment the compiler.

This one is much easier cause statements are easy to understand, but will reach to the point where expressions are not easy to understand, for instance `ConvertElementAccessCoalesceAssignment`.

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/WhileStatement.cs [hunk 1]
```diff
@@ -16,6 +16,30 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/WhileStatement.cs:20]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/TryStatement.cs [hunk 2]
```diff
@@ -17,6 +17,38 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/TryStatement.cs:21]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/ThrowStatement.cs [hunk 3]
```diff
@@ -13,6 +13,27 @@
 
 namespace Neo.Compiler
 {
+
+    /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/ThrowStatement.cs:17]:
```suggestion
    /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/SwitchStatement.cs [hunk 4]
```diff
@@ -17,6 +17,39 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/SwitchStatement.cs:21]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/ReturnStatement.cs [hunk 5]
```diff
@@ -16,6 +16,28 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/ReturnStatement.cs:20]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/LocalDeclarationStatement.cs [hunk 6]
```diff
@@ -17,6 +17,27 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/LocalDeclarationStatement.cs:21]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/LabeledStatement.cs [hunk 7]
```diff
@@ -17,6 +17,31 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/LabeledStatement.cs:21]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/GotoStatement.cs [hunk 8]
```diff
@@ -18,6 +18,43 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/GotoStatement.cs:22]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/ForStatement.cs [hunk 9]
```diff
@@ -18,6 +18,32 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/ForStatement.cs:22]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/ExpressionStatement.cs [hunk 10]
```diff
@@ -17,6 +17,28 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/ExpressionStatement.cs:21]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/DoStatement.cs [hunk 11]
```diff
@@ -16,6 +16,32 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/DoStatement.cs:20]:
```suggestion
        /// <summary>
```

File: src/Neo.Compiler.CSharp/MethodConvert/Statement/ContinueStatement.cs [hunk 12]
```diff
@@ -15,6 +15,31 @@ namespace Neo.Compiler
 {
     partial class MethodConvert
     {
+
+        /// <summary>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Statement/ContinueStatement.cs:19]:
```suggestion
        /// <summary>
```

**Review by shargon — COMMENTED** [review]
Clean empty lines

**Review by shargon — APPROVED** [review]

**Comment by Jim8y** [comment]
So many empty lines...... it must be the dotnet format

**Pull request merged**